backend/tracim_backend/tests/functional/test_contents.py

```diff
@@ -1697,6 +1697,7 @@ def test_api__get_file__ok_200__nominal_case(self) -> None:
         assert content['filename'] == 'Test_file.txt'
         assert content['page_nb'] == 1
         assert content['pdf_available'] is True
+        assert content['jpeg_available'] is True
```

Comment:
rename to `has_jpeg_preview` (by the way pdf_available must be renamed too)

If you have a prefix `is_` or `has_`, it makes explicite in all cases that it is a boolean

You must fix this everywhere in the code.

Reply:
done.